sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 3.0.0-beta.1 (Unreleased)
-
+- Update `com.azure` group id to `com.azure.spring`.
```

Comment:
This should be in a Breaking Changes section and the artifact ID change should also be addressed.